src/SearchResults.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import SearchDetails from "./SearchDetails";
+import CustomerProfile from "./CustomerProfile";
+
+function SearchResults(props) {
+  const [profile, setProfile] = useState(null);
+  const [sort, setSort] = useState("DEFAULT");
+  const [sortedResults, setSortedResults] = useState(props.results);
+
+  useEffect(() => {
+    console.log(sort);
+    let updatedResults = [...props.results].sort((a, b) => {
+      if (sort === "ASC") {
+        if (a.firstName > b.firstName) {
+          return -1;
+        } else if (a.firstName < b.firstName) {
+          return 1;
+        }
+      } else if (sort === "DESC") {
+        if (a.firstName > b.firstName) {
+          return 1;
+        } else if (a.firstName < b.firstName) {
+          return -1;
+        }
+      } else {
+        return 0;
+      }
```

Comment:
There is a lot of logic here, to make it a bit clearer it might be a good idea to separate this into a separate function